dockerfiles/vcpkg/build.sh

```diff
@@ -24,24 +29,25 @@ echo "DOCKER_PASSWORD: $DOCKER_PASSWORD"
 
 docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
 
+
```

Comment:
Remove blank line for cinsistency

Reply:
Removed and checked the rest of the file. Should only have 1 line breaks in there now.